spanner/ddl/ast.go

```diff
@@ -94,15 +94,16 @@ var PGSQL_TO_STANDARD_TYPE_TYPEMAP = map[string]string{
 	PGJSONB:       JSON,
 }
 
-// List taken from - https://www.postgresql.org/docs/current/sql-keywords-appendix.html
+// PGDialect keyword list
```

Comment:
Can you add the source ?

Reply:
Its an internal code source link so I skipped adding it here.